File: SarfMalzemeStok.Domain/Model/ProductInformation.cs
using Abp.Domain.Entities;
using System;
using System.Collections.Generic;
using System.Text;

namespace SarfMalzemeStok.Domain.Model
{
    public class ProductInformation: Entity
    {
        public int MaterialId { get; set; }
        public int MalGirisiIslemeSuresi { get; set; }
        public double ToplamAcikSiparisMiktari { get; set; }
        public double AcikSATMiktari { get; set; }
        public DateTime ImalataSonVerilisTarihi { get; set; }
        public int DahiliUretimSuresi { get; set; }
        public double ToplamAcikSiparisTutari { get; set; }

        public Material material { get; set; }

    }
}

File: SarfMalzemeStok/ClientApp/src/app/models/productInformation.ts
import { Material } from './material';

export class ProductInformation {
    id: number;
    materialId: number;
    malGirisiIslemeSuresi: number;
    toplamAcikSiparisMiktari: number;
    acikSATMiktari: number;
    imalataSonVerilisTarihi: Date;
    dahiliUretimSuresi: number;
    toplamAcikSiparisTutari: number;
    material: Material;
}
